elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcSubsystemTransformers.java

```diff
@@ -62,4 +64,23 @@ private static void from3(ChainedTransformationDescriptionBuilder chainedBuilder
                 .end();
     }
 
+    // TODO: Figure out how to handle the transformer with a preview level attribute being added to the subsystem
```

Comment:
@PrarthonaPaul I think we can update this now following a similar approach that was used for the scope RFE, right?

Reply:
I think so. 
The transofrmer tests are failing for me now, likely because of the new model and schema versions added. 
Once I figure that out, I will push transformer changes. 
Should I update the transformer tests to check for transformation using the request related attributes instead of scope now? or leave it as is?

Reply:
We need to introduce a new `from4` method, i.e., from 4.0.0 (WildFly 33) to 3.0.0 (WildFly32), that will handle your new request related attributes and then add corresponding tests as well. The existing `from3` method will be left unchanged.

Reply:
I think this is still pending, right?

Reply:
This has been implemented.